mkdocs.yml

```diff
@@ -8,44 +8,50 @@ theme:
   logo: resource/pics/darwinia-logo.png
   favicon: resource/pics/background.png
   font:
-      text: Open Sans
-      code: Roboto Mono
+    text: San Francisco
+    code: SFMono-Regular
```

Comment:
Please don't change the font. 

I don't think the new one is better than the current one. Or you can give me something more convincing.

Reply:
I would vote for new font as it looks more clear in my brower.

<img width="1201" alt="image" src="https://github.com/darwinia-network/document/assets/1070122/a6465cdf-d0cb-4bb0-a69c-504eef3a39ba">